index.html

```diff
@@ -643,12 +643,15 @@ <h2>Security</h2>
           </ul>
         </div>
         <p><span class="rfc2119-assertion" id="common-constraints-security-2">
-            Conformant Consumers MUST support all of these security schemes.</span>
+            Conformant Consumers MUST support at least all of these security schemes.</span>
         </p>
 
         <p><span class="rfc2119-assertion" id="common-constraints-security-3">
             A Thing MAY implement multiple security schemes.</span>
         </p>
+        <p><span class="rfc2119-assertion" id="common-constraints-security-4">
+            A Thing MUST support at least one of the above security schemes.</span>
```

Comment:
I'm sorry for asking to keep asking for further changes but there are now two similar sounding assertions which say different Things:

> Below is a list of security schemes which conformant Web Things MAY use

> A Thing MUST support at least one of the above security schemes.

The text originally said "Below is a list of security schemes which conformant Web Things MAY use and conformant Consumers MUST support" but I believe it was split into two assertions because sentences can't include two RFC 2119 keywords.

The former assertion also uses the term "MAY" incorrectly, which is my fault.

Let's just stop dancing around grammatical structures and duplicate the list 🙂...

Conformant Web Things MUST use at least one of the following security schemes [[wot-thing-description]]:
- NoSecurityScheme
- BasicSecurityScheme
- OAuth2SecurityScheme with the code or client flow.

Conformant Consumers MUST support all of the following security schemes [[wot-thing-description]]:
- NoSecurityScheme
- BasicSecurityScheme
- OAuth2SecurityScheme with the code or client flow.